frontend/src/Editor/Inspector/Components/Table/Table.jsx

```diff
@@ -923,7 +923,10 @@ class TableComponent extends React.Component {
     const useDynamicColumn = component.component.definition.properties.useDynamicColumn?.value
       ? resolveReferences(component.component.definition.properties.useDynamicColumn?.value, currentState) ?? false
       : false;
-
+    const allowSelection = component.component.definition.properties?.allowSelection?.value
+      ? resolveReferences(component.component.definition.properties.allowSelection?.value, currentState)
+      : resolveReferences(component.component.definition.properties.highlightSelectedRow.value, currentState) ||
+        resolveReferences(component.component.definition.properties.showBulkSelector.value, currentState);
```

Comment:
Instead of ternary we can use nullish coalescing. If the value of `allowSelection` is null then it will go to else statement
```suggestion
    const allowSelection = component.component.definition.properties?.allowSelection?.value
      ?? resolveReferences(component.component.definition.properties.highlightSelectedRow.value, currentState) ||
        resolveReferences(component.component.definition.properties.showBulkSelector.value, currentState);
```